common/app/routes/challenges/components/project/Side-Panel.jsx

```diff
@@ -33,11 +34,19 @@ export default class SidePanel extends PureComponent {
   }
 
   render() {
-    const { title, description, isCompleted } = this.props;
+    const { title, description, isCompleted, blockType } = this.props;
+    let titleString = () => {
```

Comment:
This logic does not need to be in a function.

This looks like repeat code. If so it can be abstracted out into a util file